check_run/check_run/doctype/check_run/check_run.py

```diff
@@ -277,8 +277,8 @@ def get_entries(doc):
 	if frappe.db.exists('Check Run', doc.name):
 		db_doc = frappe.get_doc('Check Run', doc.name)
 		print("dirty", db_doc.is_dirty(doc))
```

Comment:
Looks like we can remove this `is_dirty` function.

Reply:
Done